tasks/go.py

```diff
@@ -154,6 +154,29 @@ def cyclo(ctx, targets, limit=15):
     print("gocyclo found no issues")
 
 
+@task
+def golangci_lint(ctx, targets, rtloader_root=None, build_tags=None):
+    """
+    Run golangci-lint on targets using .golangci.yml configuration.
+
+    Example invokation:
+        inv golangci_lint --targets=./pkg/collector/check,./pkg/aggregator
```

Comment:
Should it be `golangci-lint` instead of `golangci_lint`

Reply:
Nit: `invokation` -> `invocation`

Reply:
Correct me if I'm wrong, `golangci_lint` task refers to this Invoke task: https://github.com/DataDog/datadog-agent/pull/4538/files#diff-ff17943812ec2171c673c7b301d38bc8R21 that uses underscore.

Reply:
Exactly `golangci_lint` is the invoke command

Reply:
invoke replace `_` by `-`, good catch !